src/components/VmDialog/sagas.js

```diff
@@ -1,21 +1,24 @@
 import { put, takeEvery, takeLatest } from 'redux-saga/effects'
-import { refresh, getSingleVm } from '../../actions/index'
+
 import { ADD_NEW_VM, EDIT_VM } from './constants'
-import { setSavedVm } from './actions'
 import Api from '../../ovirtapi'
+import { callExternalAction } from '../../saga/utils'
+import { fetchSingleVm } from '../../sagas'
+import { refresh, getSingleVm } from '../../actions'
+import { setSavedVm } from './actions'
 
-function* createNewVm (sagas, action) {
-  const result = yield sagas.callExternalAction('addNewVm', Api.addNewVm, action)
+function* createNewVm (action) {
+  const result = yield callExternalAction('addNewVm', Api.addNewVm, action)
   if (!result.error) {
     yield put(refresh({ page: action.payload.page }))
     yield put(setSavedVm({ vm: Api.vmToInternal({ vm: result }) }))
   }
 }
 
-function* editVm (sagas, action) {
-  const result = yield sagas.callExternalAction('editVm', Api.editVm, action)
+function* editVm (action) {
+  const result = yield callExternalAction('editVm', Api.editVm, action)
   if (!result.error) {
-    const result2 = yield sagas.callExternalAction('changeCD', Api.changeCD, {
```

Comment:
I think it could be named more specific.

Reply:
I was gonna say the same, but it was already named that, so we're good